psql2mysql/__init__.py

```diff
@@ -289,6 +322,15 @@ def add_subcommands(subparsers):
     parser = subparsers.add_parser(
         'migrate',
         help='Migrate data from PostgreSQL to MariaDB')
+
+    parser.add_argument("--chunked",
+                        action='store_true',
+                        default=False,
```

Comment:
Given the MariaDB constraints that we have about writing records, I think this should be on by default.